packages/utils/src/get-snark-artifacts/config.ts

```diff
@@ -0,0 +1,8 @@
+import { ArtifactType, ProofType } from "../types"
+
+export const ARTIFACTS_BASE_URL = "https://zkkit.cedoor.dev"
+export const ARTIFACTS_TYPES = [ArtifactType.WASM, ArtifactType.ZKEY]
+export const ARTIFACT_BASE_URLS: Record<ProofType, string> = {
+    [ProofType.POSEIDON]: `${ARTIFACTS_BASE_URL}/${ProofType.POSEIDON}-proof/artifacts`,
```

Comment:
Semaphore is also using the same functions: https://github.com/semaphore-protocol/semaphore/blob/main/packages/proof/src/get-snark-artifacts.node.ts

We could add support for it here as well, maybe using the same ZK-Kit base URL (?)

Reply:
good idea. Will do it